src/frontends/pytorch/src/op/topk.cpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <climits>
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/opsets/opset10.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+OutputVector translate_topk(NodeContext& context) {
+    const auto input_tensor = context.get_input(0);
+    const auto largest = context.const_input<bool>(3);
+    const auto sorted = context.const_input<bool>(4);
+    auto k = context.get_input(1);
+    int64_t axis{-1};
+    auto mode = opset10::TopK::Mode::MAX;
+    auto sort = opset10::TopK::SortType::SORT_VALUES;
+
+    if (!context.input_is_none(2)) {
+        axis = context.const_input<int64_t>(2);
+    }
+    if (!largest) {
```

Comment:
nit: The logic for this seems backward. It would be better to have `mode` and `sort` be defined as default first and if `largest` or `sorted` is provided modify it.

Reply:
okok, I thought I will follow the default values from Pytorch.